pkg/ctxutil/ctxutil.go

```diff
@@ -251,17 +258,61 @@ func IsForce(ctx context.Context) bool {
 	return is(ctx, ctxKeyForce, false)
 }
 
-// WithCommitMessage returns a context with a commit message set.
+// WithMultiSecretOperation returns a context with a flag set
+func WithMultiSecretOperation (ctx context.Context, bv bool) context.Context {
+	return context.WithValue(ctx, ctxKeyIsMultiSecretOperation, bv)
+}
+
+// IsMultiSecretOperation returns true if the flag is set to true
+func IsMultiSecretOperation (ctx context.Context) bool {
+	if hasBool(ctx,ctxKeyIsMultiSecretOperation) {
+		return is(ctx, ctxKeyForce, true)
+	} else {
+		return false
+	}
+}
+
+
+// AddToCommitMessageBody returns a context with something added to the commit's body
+func AddToCommitMessageBody(ctx context.Context, sv string) context.Context {
```

Comment:
I'm not super happy with this approach. It should work, but it's neither elegant nor efficient.

What you're doing is:
- get an existing string from the context (if remains in the content)
- concatenate it to create a new strings
- add that to the context (the other value is still there)

In most cases this shouldn't be a big deal, but I can imagine very big password stores where this might become an issue.

So what about another approach: Create a new commit message type (either a custom type with a strings.Builder/bytes.Buffer) or just a ref to one of those. If we only store the reference in the context we can manipulate the value and don't need to add it again on every single invocation.